Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/BaseManipulator.h

```diff
@@ -300,6 +301,7 @@ namespace AzToolsFramework
         void SetLocalPosition(const AZ::Vector3& localPosition);
         void SetLocalOrientation(const AZ::Quaternion& localOrientation);
         void SetNonUniformScale(const AZ::Vector3& nonUniformScale);
+        virtual AZStd::vector<BaseManipulator*> GetManipulators() = 0;
```

Comment:
Also this type doesn't really have any virtual functions (other than the slightly awkward `RefreshView` which some need), so it would be great if we don't have to add another and can solve this in a different way